### DISCUSSION TITLE
ClassNotFoundException  Resource_desktopKt

### DISCUSSION THREAD

**Discussion by andreshl** [KevinnZou/compose-webview-multiplatform#117] [Q&A]
hello, i try to load a local html file in the webview but i receive this error when runs in dekstop with the version 1.9.0

image added,
![image](https://github.com/KevinnZou/compose-webview-multiplatform/assets/17575329/daaee391-b615-417f-be54-ba3b9567034b)

my gradle

[gradle.txt](https://github.com/KevinnZou/compose-webview-multiplatform/files/14534181/gradle.txt)

¿someone can helpme ?

**Comment by KevinnZou** [comment]
@andreshl Thanks for your feedback! The reason is that you are using compose multiplatform 1.6.0, which has switched to a new method of resource management. Unfortunately, this library does not currently support it. However, I am planning to add support for it this month. Please wait for the latest release.

**Reply by andreshl** [reply]
Thanks, i wait patiently the versión. 
I want to investigate More i'm really interested in the jsbridge AND load local html's in multiplatform environments.

Thanks for your work.

**Reply by KevinnZou** [reply]
Thank you for your patience!
